xml/System.Runtime.Intrinsics.X86/Aes.xml

```diff
@@ -40,7 +42,10 @@
       <Docs>
         <param name="value">To be added.</param>
         <param name="roundKey">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>
+            __m128i _mm_aesdec_si128 (__m128i a, __m128i RoundKey)
```

Comment:
are these things the name of the instructions?

Reply:
The first line (generally containing `_mm_*`) are the method signatures for the C/C++ version of these intrinsics.

The second line (`AESDEC xmm, xmm/m128` in this case) is the assembly instruction.